ambari-agent/conf/unix/install-helper.sh

```diff
@@ -41,11 +41,11 @@ LOG_FILE=/dev/null
 
 CLEANUP_MODULES="resource_management;ambari_commons;ambari_agent;ambari_ws4py;ambari_stomp;ambari_jinja2;ambari_simplejson"
 
-OLD_COMMON_DIR="/usr/lib/python2.6/site-packages/ambari_commons"
-OLD_RESOURCE_MANAGEMENT_DIR="/usr/lib/python2.6/site-packages/resource_management"
-OLD_JINJA_DIR="/usr/lib/python2.6/site-packages/ambari_jinja2"
-OLD_SIMPLEJSON_DIR="/usr/lib/python2.6/site-packages/ambari_simplejson"
-OLD_AMBARI_AGENT_DIR="/usr/lib/python2.6/site-packages/ambari_agent"
+OLD_COMMON_DIR="/usr/lib/python3.9/site-packages/ambari_commons"
+OLD_RESOURCE_MANAGEMENT_DIR="/usr/lib/python3.9/site-packages/resource_management"
+OLD_JINJA_DIR="/usr/lib/python3.9/site-packages/ambari_jinja2"
+OLD_SIMPLEJSON_DIR="/usr/lib/python3.9/site-packages/ambari_simplejson"
+OLD_AMBARI_AGENT_DIR="/usr/lib/python3.9/site-packages/ambari_agent"
```

Comment:
Can we extract to one variable so that we can change the python version in one place..?

Reply:
Generate a build script build-ambari.sh to solve the problem. related link: https://github.com/apache/ambari/pull/3681/commits/6e8082a3a9c133ed40df603783113a9e8c83fce4

Reply:
I feel like Ambari should come with a venv for its own packages rather than pollute OS-level site-packages